drivebrain_core_impl/drivebrain_control/src/SimpleController.cpp

```diff
@@ -39,7 +34,7 @@ bool control::SimpleController::init()
     std::optional max_regen_torque = get_live_parameter<torque_nm>("max_regen_torque");
     std::optional rear_torque_scale = get_live_parameter<float>("rear_torque_scale");
     std::optional regen_torque_scale = get_live_parameter<float>("regen_torque_scale");
-    std::optional positive_speed_set = get_live_parameter<speed_m_s>("positive_speed_set");
+    std::optional positive_speed_set = get_parameter_value<speed_m_s>("positive_speed_set");
```

Comment:
why is this no longer a live parameter?

Reply:
When I was creating the unit test I thought you told me to make the speed_set (which just refers to the max) constant.

Reply:
Oh, well can we put it back to live, we often use this live now